src/components/LogoLink.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts"></script>
+
+<template>
+  <div class="logo">
+    <a href="/"><strong>netflix</strong>roulette</a>
```

Comment:
You should use embedded <router-link></router-link> component instead of <a> tag, for navigation inside the app